### DISCUSSION TITLE
Accidentally deleted my project git repo but the deployment is still on vercel.

### DISCUSSION THREAD

**Discussion by CodeWizarz** [vercel/community#3633] [Help] [answered]
### Summary

Hey y'all i deployed my project on vercel and accidentally deleted my repo in github unable to recover the repo. Tried many scripts to download source using vercel api none of them worked. I'm able to see my folder structure on my project Source tab in vercel. Any idea on how to retrieve the source from vercel

### Steps to Reproduce

**

**Comment by swarnava** [comment, marked as answer]
Hi, 

We don't store source code in our server so you won't be able to export them from our server. You can contact to your source code management platform support for further help to recover this.
